src/main/java/com/training/web/pages/flipkart/FlipkartHomePage.java

```diff
@@ -25,4 +28,32 @@ public ResultPage clickSearch() {
         $x(submit).shouldBe(visible).click();
         return new ResultPage();
     }
+
+    public String clickSocialMediaPage(String media) throws InterruptedException {
+
+        switch (media) {
+            case "Facebook":
+                SelenideElement facebook = $x("//a[text()='Facebook']");
+                facebook.shouldBe(visible).scrollIntoView(true);
+                logger.info(facebook.getText());
+                facebook.click();
+                $x("//u[text()='Facebook']").shouldBe(enabled);
+                break;
+            case "Twitter":
+                SelenideElement twitter = $x("//a[text()='Twitter']");
+                twitter.shouldBe(visible).scrollIntoView(true);
+                logger.info(twitter.getText());
+                twitter.click();
+                $x("//a[@aria-label='Twitter']").shouldBe(visible);
+                break;
+            case "YouTube":
+                SelenideElement youtube = $x("//a[text()='YouTube']").shouldBe(visible);
+                youtube.shouldBe(visible).scrollIntoView(true);
+                logger.info(youtube.getText());
+                youtube.click();
+                $x("//paper-button[@aria-label='Subscribe']").shouldBe(visible);
+                break;
```

Comment:
From line 36 to 40 same LOC is repeated, can you create one method and put that repeated  LOC and define it globally.